Manifests/ManagedPreferencesApplications/com.secondsonconsulting.renew.plist

```diff
@@ -0,0 +1,543 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	
+	<!-- *************************** OPTIONAL KEYS *************************** -->
```

Comment:
Once you're done creating your manifest, you can remove all of the XML comments.

Reply:
Removed all XML comments aside from those indicating which fields should be updated, I think those would be useful to keep in the file for future reference. If you would prefer I remove those as well, let me know.

Reply:
Yes, please remove those too. See #567 to understand why. Thanks 🙂

Reply:
Thank you for the link, I appreciate the thoroughness